### DISCUSSION TITLE
https://www.aissquare.com/models/detail?name=DPA_v2.0.0&id=150&pageType=models

### DISCUSSION THREAD

**Discussion by giscus** [deepmodeling/AIS-Square#218] [Announcements]
# https://www.aissquare.com/models/detail?name=DPA_v2.0.0&id=150&pageType=models

https://www.aissquare.com/models/detail?name=DPA_v2.0.0&id=150&pageType=models

**Comment by Mile-Away** [comment]
请问这个模型条目如何使用，有相关的教程吗～ 谢谢 🙏

**Reply by Andy6M** [reply]
同问，请问可以在deepmdv3.0.0a0上进行微调吗？对应的input.json版本是哪个，谢谢！

**Reply by iProzd** [reply]
目前这个模型还只能在之前的代码上进行微调，是 https://doi.org/10.5281/zenodo.10428497 链接中的代码；适用于deepmdv3.0.0a0 的模型最近会发布，从头训练deepmdv3.0.0a0的DPA2的例子可以见https://github.com/deepmodeling/deepmd-kit/blob/devel/examples/water/dpa2/input_torch.json ，到时候微调也是用这个input.json.
